apps/src/blockly/addons/contextMenu.js

```diff
@@ -148,21 +153,59 @@ const registerKeyboardNavigation = function () {
     },
     callback: function (scope) {
       const controller = Blockly.navigationController;
-      scope.workspace.keyboardAccessibilityMode
-        ? controller.disable(scope.workspace)
-        : controller.enable(scope.workspace);
+      if (scope.workspace.keyboardAccessibilityMode) {
```

Comment:
nit: It might be work creating a couple of helpers here to improve readability, like:
```
const isAccessibilityModeEnabled = (scope) => scope.workspace.keyboardAccessibilityMode;
const isNavigationEnabled = () => experiments.isEnabled(experiments.KEYBOARD_NAVIGATION);
```
But no strong feelings either way.